### DISCUSSION TITLE
Splitting date and time when you need also do sorts on date and time

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1244] [Data Modeling]
I could save alot of space splitting date and time.  I always split date out.  However i often leave date/time also.
The reason i have not found a way sort a table on 2 columns
Think of a details table where you need to sort by datetime desc.
If i remove it i can only sort by date.
It seems like it would be an expensive operation to try and put it back together.

It really have to be sort on up to 4 columns if split
Date
Hour
Min
Sec

**Comment by otykier** [comment]
It is possible to sort the table and matrix visuals in Power BI by two or more columns by shift-clicking on the column header.

**Reply by TinyTimothy2** [reply]
wow. thank you.  not one person in our org was familiar with this!
to bad you can't do with the sort flyout panel
